lib/ansible/modules/cloud/google/gcp_bigquery_dataset_info.py

```diff
@@ -29,9 +29,11 @@
 
 DOCUMENTATION = '''
 ---
-module: gcp_bigquery_dataset_facts
+module: gcp_bigquery_dataset_info
 description:
 - Gather facts for GCP Dataset
+- This module was previously called gcp_bigquery_dataset_facts before Ansible 2.9.
+  The usage has not changed
```

Comment:
Please add a period to the end of each line with this pattern (The usage has not changed.).